VMEncryption/main/handle.py

```diff
@@ -203,7 +203,9 @@ def stamp_disks_with_settings(items_to_encrypt, encryption_config):
                         filenames.append(str(tag["Value"]))
 
     for filename in filenames:
-        shutil.copy(current_passphrase_file, os.path.join(CommonVariables.encryption_key_mount_point, filename))
+        filepath = os.path.join(CommonVariables.encryption_key_mount_point, filename)
```

Comment:
When we pull filename from the DiskEncryptionKeyFileName tag, will it ever contain slashes?  The way os.path.join works, if subsequent parameters after the root contain a slash, the root is ignored and the join may not work as anticipated.  ( see https://stackoverflow.com/a/1945930 )

Reply:
The tag should never have slashes, so we probably don't need to be worried about this.